docs/validators/key-management/multikey-nodes.md

```diff
@@ -4,58 +4,75 @@ title: Multikey nodes management
 ---
 
 [comment]: # (mx-abstract)
-This page contains information about how to manage multiple keys on an observing squad.
+This page contains information about how to manage multiple keys on a group of nodes.
 
 [comment]: # (mx-context-auto)
 ## Multikey architecture overview
 
-At the mainnet launch, and up until the release candidate RC/v1.6.0 becomes mainstream, each node could have managed only
-one key. So the relationship between the nodes and keys is `1:1`, the node "following" the shard where the managed key is assigned.
-The main idea behind the multikey feature is that a set of keys can be managed by an observing squad set. The same set of keys
-should be provided for all observing squad nodes, so the nodes can **pick** any key from that set and use it to propose or sign
-blocks whenever the consensus group in a round contains at least one managed key. The observing node that runs in multikey mode
-is able to use the provided keys in any combination possible. So one node can use one key to propose a block and one or more other
-keys to already sign on that proposed block automatically to improve performance.
-So, with the multikey feature, the relationship now becomes `n:[num_shards]`, providing that `n` is the number of keys 
-managed by an entity.
+Since the mainnet launch, and up until the release candidate RC/v1.6.0, each node could have managed only
+one key. So the relationship between the nodes and staked validator keys is `1:1`, the node "following" the shard where 
+the managed key is assigned.
+The multikey feature allows a node instance to hold more than one key. However, since MultiversX is a sharded blockchain 
+and a single node is only able to store the state of a single shard at a time, we need a group of nodes with exactly
+one node in each shard, similar with what we have on observing squad. Also, in each epoch, the keys can be shuffled among shards. 
+This means that running multiple keys will require at least the number of shards + 1 node instances (one for each shard + metachain).
+The same set of keys should be provided for all node instances. This type of nodes used in multikey operation can be assimilated
+as a hybrid between an observer node and a validator. The reason behind this affirmation is synthesized below:
+- the observer characteristic is defined by the fact that the node will hold in the `validatorKey.pem` a BLS key that will never 
```

Comment:
- the observer characteristic is defined by the fact that the node will hold ...
 =>
It behaves as an observer by holding ...

Reply:
done